### PR TITLE
Proposed Gov 12.1.2 amendment - GOV10.1.2.3 edit

### PR DESCRIPTION
**Pull request by 0xldr**
Do not merge unless poll passes.

Adds changes described here: https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal-gov10-2-3-1a-edit/24005

### CODE REVIEW

File: MIP113/MIP113.md [hunk 1]
```diff
@@ -814,11 +814,25 @@ The LINE_MOM contract manages the breaker for the Debt Ceilings of a configurabl
...
+The whitelist may be configured via a successful governance proposal, but must wait the GSM Delay before changes come into effect. The whitelist is defined in *10.2.3.1A* and can be changed via the weekly governance cycle
 
-- PSM (USDC) - PSM-USDC-A
-- PSM (PAX) - PSM-PAX-A
-- PSM (GUSD) - PSM-GUSD-A
+##### 10.2.3.1A: Debt Ceiling Breaker Vault Types Whitelist
+
+¤¤¤
+
+* PSM (USDC) - PSM-USDC-A
```
Comment (blimpa) [inline MIP113/MIP113.md]: (The rest of the document uses "-" for lists, not "*")

File: MIP113/MIP113.md [hunk 2]
```diff
@@ -814,11 +814,25 @@ The LINE_MOM contract manages the breaker for the Debt Ceilings of a configurabl
 
 The Debt Ceiling Breaker affects both the Debt Ceiling and the Maximum Debt Ceiling of a given vault type when activated, disabling the Dynamic Debt Ceiling functionality for that vault type if enabled. To reverse the effect, parameters of affected vault types must be reconfigured with an executive vote which is subject to GSM Pause Delay.
 
-The whitelist may be configured via a successful governance proposal, but must wait the GSM Delay before changes come into effect. At time of writing, the following vault types are whitelisted on the Debt Ceiling Breaker:
+The whitelist may be configured via a successful governance proposal, but must wait the GSM Delay before changes come into effect. The whitelist is defined in *10.2.3.1A* and can be changed via the weekly governance cycle
```
Comment (blimpa) [inline MIP113/MIP113.md]:
```suggestion
The whitelist may be configured via a successful governance proposal, but must wait the GSM Delay before changes come into effect. The whitelist is defined in *10.2.3.1A* and can be changed via the Weekly Governance Cycle.
```

File: MIP113/MIP113.md (hunk 1 above)
Comment (0xldr) [reply MIP113/MIP113.md]: We will change this if the poll closes in favor of this update.

Comment (0xldr) [reply MIP113/MIP113.md]: @blimpa please add this to the style guide

**Comment by 0xldr** [comment]
vote passed - https://vote.makerdao.com/polling/QmZsAM36

**Pull request merged**